docs/tutorials/installation.md

```diff
@@ -87,18 +99,30 @@ Marin runs on multiple types of hardware (CPU, GPU, TPU).
          ```bash
          nvcc --version
          ```
-         Finally we'll install the correct libraries for GPU setup:
+         Finally install Python deps for GPU setup:
 
          ```bash
+         # Recommended
+         uv sync --extra=cuda12
+
+         # Alternative (pip-style)
          uv pip install -e ".[cuda12]"
```

Comment:
Nit, let's just remove the pip style for all of these unless we have a good reason to keep it.